webcomponents-loader.js

```diff
@@ -23,26 +23,30 @@
   if (!window.customElements || window.customElements.forcePolyfill) {
     polyfills.push('ce');
   }
-  if (!('content' in document.createElement('template')) || !window.Promise ||
+  // NOTE: any browser that does not have template or ES6 features
+  // must load the full suite (called `lite` for legacy reasons) of polyfills.
+  if (!('content' in document.createElement('template')) || !window.Promise || !Array.from ||
     // Edge has broken fragment cloning which means you cannot clone template.content
     !(document.createDocumentFragment().cloneNode() instanceof DocumentFragment)) {
-    polyfills.push('pf');
-  }
-
-  if (polyfills.length === 4) { // hi-ce-sd-pf is actually called lite.
     polyfills = ['lite'];
   }
 
   // NOTE: we stub HTMLImports to enable WCT to wait for "WebComponentsReady" event.
-  window['HTMLImports'] = {};
```

Comment:
This will be run also when we force the polyfills, so also in Chrome >= 55 :/
I don't know how worth it is, but what if we document.write an import instead of a script here?